cgroup/cgroup.go

```diff
@@ -76,26 +63,23 @@ func (m cgroup) generatePath(kind string) (string, error) {
 
 // Read reads the given cgroup file data and returns the content as a string
 func (m cgroup) Read(kind, file string) (string, error) {
-	kindPath, err := m.generatePath(kind)
+	manager := *m.manager
+	kindPath := manager.Path(kind)
```

Comment:
note that I'm configuring this with `/proc/1/root/sys/fs/cgroup` as root cgroup path

I notice that we have configured root cgroup as `/mnt/cgroup/`, but I have no idea why. 

changing this isn't hard, but hope to gain some insight from code reviewers 🙇 .